main.go

```diff
@@ -96,30 +116,11 @@ func main() {
 		r.Use(cors.New(corsConfig))
 	}
 
-	healthModule{}.Register(r)
+  healthModule{}.DeprecatedRegister(r)
+  healthModule{}.Register(r.Group("/api"))
```

Comment:
nit: Invalid indentation

```suggestion
	healthModule{}.DeprecatedRegister(r)
	healthModule{}.Register(r.Group("/api"))
```